content/swift/concepts/arrays/terms/first/first.md

```diff
@@ -12,31 +12,29 @@ CatalogContent:
   - 'paths/build-ios-apps-with-swiftui'
 ---
 
-The **`.first()`** method returns the first element of the array that fulfills the condition, or `nil` if there is no element that fulfills the condition.
+The **`.first()`** method returns the first element of the array that fulfills the condition. Otherwise, it returns `nil`.
 
-> **Note:** `.first()` is an method while `.first` is a property. For details about the `.first` array property see the bottom part of this document.
+> **Note:** `.first()` is a method while `.first` is a property. Details about the `.first` array property can be found near the bottom of this entry.
 
 ## Syntax
 
 ```pseudo
 arrayName.first(where: { condition })
 ```
 
-The `.first()` method takes one argument:
-
-- `condition`, a [closure](https://www.codecademy.com/resources/docs/swift/closures) that takes an element of the array as its argument and returns a Boolean value indicating whether the element is a match.
+The `.first()` method takes a single `condition` parameter. It is a [closure](https://www.codecademy.com/resources/docs/swift/closures) that returns a boolean value indicating whether a given array element is a match.
```

Comment:
It would be a good idea to link out "boolean" which just happens to be the docs' concept of the week!

```suggestion
The `.first()` method takes a single `condition` parameter. It is a [closure](https://www.codecademy.com/resources/docs/swift/closures) that returns a [boolean](https://www.codecademy.com/resources/docs/general/boolean) value indicating whether a given array element is a match.
```